wiki/osu!stream/id.md

```diff
@@ -1,213 +1,310 @@
 
 **Hold-circle**
 
-- Penentuan nilai hold circle bergantung pada ketukan awal dan waktu menahan sebelum "meledak".
-  - 300 apabila kamu melakukannya secara sempurna
+- Penentuan nilai *hold circle* bergantung pada ketukan awal dan waktu menahan sebelum "meledak".
+  - 300 apabila kamu melakukannya secara sempurna.
   - 100 aoabila ketukan awalnya kurang akurat, namun menahannya sampai "meledak".
-  - 50 apabila kamu setidaknya hanya menyentuh hold circle tersebut.
-  - 30 untuk setiap ketukan.
-  - Pengali kombo akan diulang dari 0 apabila kamu tidak menahannya sampai "meledak", dan kamu mendapatkan "Miss" apabila kamu tidak menyentuhnya sama sekali.
+  - 50 apabila kamu setidaknya hanya menyentuh *hold circle* tersebut.
+  - 30 untuk setiap "ledakan".
+  - Pengali kombo akan diulang dari 0 apabila kamu tidak menahannya sampai "meledak", dan kamu mendapatkan "*miss*" apabila kamu tidak menyentuhnya sama sekali.
 
 **Slider**
 
 - **Penentuan nilai slider bergantung pada ketepatan ketukan awal.**.
-- Slider terdiri dari slider tick, yang termasuk awal, akhir, dan repeat point dari slider.
-- Kamu mendapatkan 300 apabila kamu mendapatka seluruh slider tick dan ketukan awal yang akurat, 100 apabila kamu mendapatkan setidaknya setengah dari jumlah slider tick dengan ketukan awal yang kurang akurat, dan 50 apabila kamu setidaknya mendapatkan 1 slider tick atau setidaknya mengetuk awal slider.
-- **Mengetuk awal slider terlalu cepat** tidak akan memicu miss, namun **akan mengulang pengali kombomu dari 0**.
-- Kamu hanya akan mendapatkan miss apabila kamu tidak melakukan apa-apa terhadap slider tersebut.
+- Slider terdiri dari *slider tick*, yang termasuk awal, akhir, dan *repeat point* dari *slider*.
+  - Kamu mendapatkan 300 apabila berhasil mendapatkan seluruh *slider tick* dan ketukan awal yang akurat.
+  - Kamu mendapatkan 100 apabila mendapatkan setidaknya setengah dari bagian *slider* dengan ketukan awal yang cukup akurat.
+  - Kamu mendapatkan 50 apabila setidaknya mendapatkan setidaknya satu *slider tick* atau setidaknya mendapat ketukan awal *slider*.
+- **Mengetuk awal slider terlalu cepat** tidak akan memicu *miss*, namun **akan mengulang pengali kombomu dari 0**.
+- Kamu hanya akan mendapatkan *miss* apabila kamu tidak melakukan apa-apa terhadap *slider* tersebut.
 
 **Spinner**
 
-- Untuk spinner, kamu akan mendapatkan 300, 100 atau 50 tergantung pada jumlah putaran yang kamu lakukan dibanding panjang spinnernya.
-- Kamu akan mendapatkan miss apabila kamu tidak mengisi penuh bar yang ada di sisi spinner.
+- Untuk *spinner*, kamu akan mendapatkan 300, 100 atau 50 tergantung pada jumlah putaran yang kamu lakukan dibanding panjang *spinner*-nya.
+- Kamu akan mendapatkan *miss* apabila kamu tidak mengisi penuh bar yang ada di sisi *spinner*.
```

Comment:
```suggestion
- Kamu akan mendapatkan miss apabila kamu gagal mengisi penuh *spinner metre* yang ada pada sisi kiri dan kanan spinner.
```

Reply:
might apply this but change "metre" to "meter" for consistency i think? i'm not sure

Reply:
osu! (including the wiki) has always been using british english for literally almost everything since the very beginning lol

hence why it's `spinner-metre.png` not `spinner-meter.png`

http://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#british-english

Reply:
yea, but the English version of this article uses "meter" so I thought of using that instead

lmk further what you think about this, i'm torn whether if i wanna use the British English spelling or not @Nivalyx